engines/scumm/scumm.cpp

```diff
@@ -2504,8 +2473,9 @@ void ScummEngine::scummLoop_handleSaveLoad() {
 
 			GUI::MessageDialog dialog(buf);
 			runDialog(dialog);
-		} else if (_saveLoadFlag == 1 && _saveLoadSlot != 0 && !_saveTemporaryState) {
-			// Display "Save successful" message, except for auto saves
+		}
+		else if (_saveLoadFlag == 1 && _saveLoadSlot != 0 && !_saveTemporaryState) {
```

Comment:
Why?

Reply:
Same as above. Not something I did willingly...